src/components/PrioSelector.vue

```diff
@@ -0,0 +1,80 @@
+<template>
+  <select
+    class="todo-priority high"
+    v-on:change="handlePriorityChange"
+    ref="priority"
+  >
+    <option value="high">High</option>
+    <option value="medium">Medium</option>
+    <option value="low">Low</option>
+  </select>
+</template>
+<script setup lang="ts">
+const handlePriorityChange = (event: Event) => {
```

Comment:
Please declar the function with the function keyword to be more visually separated from the variables